api/CanvasRenderingContext2D.json

```diff
@@ -2369,9 +2390,16 @@
                 "prefix": "webkit"
               }
             ],
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome_android": [
+              {
+                "version_added": "30"
+              },
+              {
+                "version_added": true,
```

Comment:
`webkitImageSmoothingEnabled` is indeed very old, added in https://trac.webkit.org/changeset/117635/webkit.

Perhaps split out `imageSmoothingEnabled` updates separately, and also ensure the Safari data is correct at the same time? At the very least it's now missing support for `webkitImageSmoothingEnabled` in Safari, but maybe it's also conflating prefixed and unprefixed.

I've sent https://github.com/foolip/mdn-bcd-collector/pull/843 to help sort this out.